src/Defaults/Trigger/Taxonomy/TermAdded.php

```diff
@@ -45,7 +45,7 @@ public function __construct( $taxonomy = 'category' ) {
 	 */
 	public function get_name() : string {
 		// Translators: taxonomy name.
-		return sprintf( __( '%s term created', 'notification' ), WpObjectHelper::get_taxonomy_name( $this->taxonomy ) );
+		return sprintf( __( '%s term created', 'notification' ), $this->taxonomy->label ?? '' );
```

Comment:
Can you check and let me know if the Trigger list in admin now works and displays the correct trigger name for taxonomies?

Reply:
<img width="900" alt="Screenshot 2022-10-21 at 21 49 18" src="https://user-images.githubusercontent.com/70862071/197277568-428fb5a2-f4d3-4418-8f50-e6ac8fa8f4ee.png">